fdbserver/MasterProxyServer.actor.cpp

```diff
@@ -329,11 +329,11 @@ ACTOR Future<Void> queueTransactionStartRequests(
 				if (req.priority >= TransactionPriority::IMMEDIATE) {
 					stats->txnSystemPriorityStartIn += req.transactionCount;
 					systemQueue->push_back(req);
-					systemQueue->span->parents.insert(req.spanContext);
+					systemQueue->span.addParent(req.spanContext);
 				} else if (req.priority >= TransactionPriority::DEFAULT) {
 					stats->txnDefaultPriorityStartIn += req.transactionCount;
 					defaultQueue->push_back(req);
-					defaultQueue->span->parents.insert(req.spanContext);
+					systemQueue->span.addParent(req.spanContext);
```

Comment:
Is the systemQueue here intended? Why it is no longer defaultQueue?

Reply:
oh good catch! This is a bug, I fixed it.